_pytest/pytester.py

```diff
@@ -22,6 +22,11 @@
 from _pytest.assertion.rewrite import AssertionRewritingHook
 
 
+IGNORE_PAM = [  # filenames added when obtaining details about the current user
```

Comment:
What's "PAM"?

Reply:
thanks for noting, pam comes frim the plugablle authentification modules, the SSS module is the one that opens this file, constant however is indeed named bdly